pkg/antctl/raw/featuregates/command.go

```diff
@@ -172,7 +172,7 @@ func output(resps []featuregates.Response, runtimeMode string, output io.Writer)
 	case runtime.ModeController:
 		output.Write([]byte("Antrea Controller Feature Gates\n"))
 	}
-	formatter := "%-25s%-15s%-10s\n"
+	formatter := "%-30s%-15s%-10s\n"
```

Comment:
Could we get the longest length of each column and use the value to determine the formatter?

Reply:
fix: https://github.com/antrea-io/antrea/pull/5314/commits/fef49d5f34b9efd95a91429601a67442bb2e394a